src/stream/src/task/barrier_manager/managed_state.rs

```diff
@@ -193,12 +182,27 @@ impl Display for ManagedBarrierStateDebugInfo<'_> {
     }
 }
 
+#[derive(Default)]
+struct ActorMutationSubscribers {
+    #[expect(clippy::type_complexity)]
+    pending_subscribers: BTreeMap<u64, Vec<mpsc::UnboundedSender<(u64, Option<Arc<Mutation>>)>>>,
+    started_subscribers: Vec<mpsc::UnboundedSender<(u64, Option<Arc<Mutation>>)>>,
```

Comment:
The type complexity here indeed looks too high... How about

```suggestion
    pending_subscribers: BTreeMap<u64, Vec<MutationSubscriber>>,
    started_subscribers: Vec<MutationSubscriber>,
```